portfolio/src/main/webapp/script.js

```diff
@@ -47,10 +47,116 @@ function addActivity() {
-  const activityTitleContainer = document.getElementById('activity-title-container');
+  const activityTitleContainer = 
+      document.getElementById('activity-title-container');
   activityTitleContainer.innerText = activityTitle;
-  const activityDescriptionContainer = document.getElementById('activity-description-container');
+  const activityDescriptionContainer = 
+      document.getElementById('activity-description-container');
   activityDescriptionContainer.innerText = activityDescription;
   const activityImageFilenameItem = document.getElementById('activity-image');
   activityImageFilenameItem.setAttribute('src', activityImageFilename);
 }
+
+/**
+ * Gets previous comments and loads them, formatted, to page
+ * @param {number=} maxComments an optional number specifying how many comemnts
+ *     to load
+ * @param {string=} sort direction by string
+ */
+function getComments(maxComments = 5, sortDirection = 'latest') {
+  fetch('/comments?max='+maxComments+'&sort='+sortDirection).then(
+        (response) => response.json()).then((commentsData) => {
+    const commentsContainer = 
+        document.getElementById('comments-list-container');
+    commentsContainer.innerHTML = '';
+
+    if (commentsData.length === 0) {
+      commentsContainer.innerHTML = 'No comments currently. Comment now!';
+    } else {
+      let i;
+      let commentsText = '';
+      for (i = 0; i < commentsData.length; i++) {
+        const commentEl = 
+            createCommentElement(commentsData[i].id, commentsData[i].name,
+                commentsData[i].commentText);
+        commentsContainer.appendChild(commentEl);
+      }
+    }
+  });
+}
+
+/**
+ * Changes number of comments displayed on page
+ */
+function changeCommentNumber() {
+  getComments(document.getElementById('comment-number').value,
+      document.getElementById('sort-direction').value);
+}
+
+/**
+ * Changes sort order of comments displayed on page
+ */
+function changeSort() {
+  getComments(document.getElementById('comment-number').value,
+      document.getElementById('sort-direction').value);
+}
+
+/**
+ * Deletes all comments from datastore by calling DeleteDataServlet post
+ */
+function deleteComments() {
+  fetch('/delete-data', {method: 'POST'}).then(window.location.reload(true));
+  document.getElementById('comment-lists-container').remove();
+}
+
+/**
+ * Deletes a given comment from datastore by calling DeleteDataServlet post
+ * @param {number} id of comment to be deleted
+ */
+function deleteComment(id) {
+  fetch('/delete-data?id='+id, {method: 'POST'}).then(
+      window.location.reload(true));
+}
+
+/**
+ * Creates formatted media list element for comment, from Bootstrap formatting
+ * @param {number} id of comment
+ * @param {string} name on comment
+ * @param {string}
+ * @return {Element} formatted element created from name and commentText
+ */
+function createCommentElement(id, name, commentText) {
+  const liElement = document.createElement('li');
```

Comment:
You could be more specific and call this commentContainer or something of that sort.